src/special/commitizen.js

```diff
@@ -0,0 +1,22 @@
+import path from 'path';
+
+export default function parseCommitizen(content, filePath, deps, rootDir) {
+  const packageJsonPath = path.resolve(rootDir, 'package.json');
+  const resolvedFilePath = path.resolve(filePath);
+
+  if (resolvedFilePath === packageJsonPath) {
+    const metadata = JSON.parse(content);
+
+    if (metadata.config && metadata.config.commitizen && metadata.config.commitizen.path) {
+      const {path} = metadata.config.commitizen;
```

Comment:
I think you will get ESLint warnings here, because `path` is already defined from `import path from 'path'`. Could you please wait and rebase on my ESLint fix? We should ensure ESLint from CI build.